kattelmod/systems/mkat/session.py

```diff
@@ -22,8 +24,15 @@ async def _get_telstate(self, args: argparse.Namespace) -> TelescopeState:
             endpoint = await self.sdp.get_telstate()
         else:
             endpoint = ''
-        return None if not endpoint else TelescopeState(endpoint) \
-            if endpoint != 'fake' else TelescopeState()
+
+        if not endpoint:
+            return None
+        elif endpoint != 'fake':
```

Comment:
Thanks for unpacking my interminable ternary!

Maybe swap the last two so that we have a nice positive `elif endpoint == 'fake':`.

Reply:
Sure, done.